Makefile

```diff
@@ -670,8 +672,34 @@ cpu_extension:
 
 run_unit_tests:
 	./prepare_llm_models.sh ${TEST_LLM_PATH}
-	./prepare_gpu_models.sh ${GPU_MODEL_PATH}
-	docker run -v $(shell realpath ./rununittests.sh):/ovms/./rununittests.sh -v $(shell realpath ${GPU_MODEL_PATH}):/ovms/src/test/face_detection_adas:ro -v $(shell realpath ${TEST_LLM_PATH}):/ovms/src/test/llm_testing:ro -e https_proxy=${https_proxy} -e RUN_TESTS=1 -e JOBS=$(JOBS) -e debug_bazel_flags=${BAZEL_DEBUG_FLAGS} $(OVMS_CPP_DOCKER_IMAGE)-build:$(OVMS_CPP_IMAGE_TAG)$(IMAGE_TAG_SUFFIX) ./rununittest.sh > test.log 2>&1 ; exit_status=$$? ; tail -200 test.log ; exit $$exit_status
+ifeq ($(RUN_GPU_TESTS),ON)
```

Comment:
does it run only GPU tests? it looks like all tests and it much be applied always with RUN_TESTS

Reply:
RUN_GPU_TESTS enables  OpenCL & VA API unit tests (enables building those and then unskipping it). This is ~10 additional unit tests which are not executed without RUN_GPU_TESTS.